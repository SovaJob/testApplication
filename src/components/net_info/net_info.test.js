// @flow


/* REACT */
import React from "react";
/* MODULES */
import renderer from "react-test-renderer";

/* CUSTOM MODULES */
import NetInfoComponent from "./net_info";


describe("NetInfoComponent tests", () => {
  test("renders online", (): void => {
    const tree = renderer
      .create(
        <NetInfoComponent />
      );

    expect(tree.toJSON()).toMatchSnapshot();
  });

});
