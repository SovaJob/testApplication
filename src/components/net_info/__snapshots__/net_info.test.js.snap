// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NetInfoComponent tests renders online 1`] = `null`;
